m2cgen/assemblers/boosting.py

```diff
@@ -84,9 +84,11 @@ class XGBoostModelAssembler(BaseBoostingAssembler):
 
     def __init__(self, model):
         feature_names = model.get_booster().feature_names
```

Comment:
Thanks for this fix!
Maybe it would be easier to make
```python
feature_names = model.get_booster().feature_names or []
```
Or maybe even better
```python
name: idx for idx, name in enumerate(feature_names or [])
```
?